File: junit5-lifecycle/src/main/java/com/arhohuttunen/junit5/lifecycle/JettyServlet.java
package com.arhohuttunen.junit5.lifecycle;

import jakarta.servlet.http.HttpServlet;
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;

public class JettyServlet extends HttpServlet {
    @Override
    protected void doGet(HttpServletRequest req, HttpServletResponse resp) {
        resp.setStatus(HttpServletResponse.SC_OK);
    }
}

File: junit5-gradle-kotlin/src/main/kotlin/com/arhohuttunen/junit5/kotlin/JettyServlet.kt
package com.arhohuttunen.junit5.kotlin

import jakarta.servlet.http.HttpServlet
import jakarta.servlet.http.HttpServletRequest
import jakarta.servlet.http.HttpServletResponse

class JettyServlet : HttpServlet() {
    override fun doGet(req: HttpServletRequest, resp: HttpServletResponse) {
        resp.status = HttpServletResponse.SC_OK
    }
}